tekton/src/pipelines/fvt-manage-health.yml.j2

```diff
@@ -1,52 +0,0 @@
----
-apiVersion: tekton.dev/v1beta1
-kind: Pipeline
-metadata:
-  name: mas-fvt-manage-health
-spec:
-  workspaces:
-    # The generated configuration files
-    - name: shared-configs
-
-  params:
-    # Tekton Pipeline image pull policy (for ibmmas/cli images)
-    - name: image_pull_policy
-      type: string
-      default: IfNotPresent
-      description: Pull policy for pipeline container images
-
-    # MAS Configuration
-    - name: mas_app_channel_manage
-      type: string
-      default: ""
-    - name: mas_instance_id
-      type: string
-      default: ""
-    - name: mas_workspace_id
-      type: string
-      default: ""
-
-    # FVT Configuration
-    - name: fvt_image_registry
-      type: string
-      default: ""
-    - name: fvt_artifactory_username
-      type: string
-      default: ""
-    - name: fvt_artifactory_token
-      type: string
-      default: ""
-
-    # Image Digests
```

Comment:
@unnati-solanki-git these digests must be transferred to the pipeline where Health tasks are, fvt-manage-is, otherwise the tasks will not have the parameters to use